controllers/host/platformnetworks_test.go

```diff
@@ -0,0 +1,646 @@
+
+			Eventually(func() bool {
+				err := k8sClient.Get(ctx, crd_key, crd_fetched)
+				if err == nil {
+					crd_fetched.ObjectMeta.Finalizers = []string{}
+					err = k8sClient.Update(ctx, crd_fetched)
+					if err == nil {
+						err = k8sClient.Delete(ctx, crd_fetched)
+						if err == nil {
+							return true
+						}
+					}
+				}
+				return false
+			}, timeout, interval).Should(BeTrue())
+
+			Eventually(func() bool {
+				err := k8sClient.Get(ctx, crd_key, crd_fetched)
+				return err != nil
+			}, timeout, interval).Should(BeTrue())
+		}
+	}
+}
+
+func DeletePlatformNetwork(nwk_name string) {
+	key := types.NamespacedName{
+		Name:      nwk_name,
+		Namespace: TestNamespace,
+	}
+
+	fetched := &starlingxv1.PlatformNetwork{}
+
+	Eventually(func() bool {
+		err := k8sClient.Get(ctx, key, fetched)
+		if err == nil {
+			fetched.ObjectMeta.Finalizers = []string{}
+			err = k8sClient.Update(ctx, fetched)
+			if err == nil {
+				err = k8sClient.Delete(ctx, fetched)
+				if err == nil {
+					return true
+				}
+			}
+		}
+		return false
+	}, timeout, interval).Should(BeTrue())
+
+	Eventually(func() bool {
+		err := k8sClient.Get(ctx, key, fetched)
+		return err != nil
+	}, timeout, interval).Should(BeTrue())
+}
+
+func DeleteAddressPool(pool_name string) {
+	key := types.NamespacedName{
+		Name:      pool_name,
+		Namespace: TestNamespace,
+	}
+
+	fetched := &starlingxv1.AddressPool{}
+
+	Eventually(func() bool {
+		err := k8sClient.Get(ctx, key, fetched)
+		if err == nil {
+			fetched.ObjectMeta.Finalizers = []string{}
+			err = k8sClient.Update(ctx, fetched)
+			if err == nil {
+				err = k8sClient.Delete(ctx, fetched)
+				if err == nil {
+					return true
+				}
+			}
+		}
+		return false
+	}, timeout, interval).Should(BeTrue())
+
+	Eventually(func() bool {
+		err := k8sClient.Get(ctx, key, fetched)
+		return err != nil
+	}, timeout, interval).Should(BeTrue())
+}
+
+func SimulateVIMStrategyAction(hostname string, expect_strategy string) {
+	host_key := types.NamespacedName{
+		Name:      hostname,
+		Namespace: TestNamespace,
+	}
+	host_instance := &starlingxv1.Host{}
+
+	Eventually(func() bool {
+		err := k8sClient.Get(ctx, host_key, host_instance)
+		Expect(err).To(BeNil())
+		return (host_instance.Status.StrategyRequired == expect_strategy)
+	}, timeout, interval).Should(BeTrue())
+
+	if host_instance.Status.StrategyRequired == cloudManager.StrategyLockRequired {
+		HostsListBodyResponse = strings.Replace(HostsListBody, `"administrative": "unlocked",`, `"administrative": "locked",`, 1)
+		HostsListBodyResponse = strings.Replace(HostsListBodyResponse, `"operational": "enabled"`, `"operational": "disabled"`, 1)
+	} else {
+		HostsListBodyResponse = HostsListBody
+	}
+}
+
+var _ = Describe("Networking utils", func() {
+
+	Context(" Verify reconfiguration is blocked for all networks in Day-1", func() {
+		It("Should be created successfully and Reconciled should be true & InSync should be 'false'", func() {
```

Comment:
Change "Should be created successfully and Reconciled should be true & InSync should be 'false'"

to

"Should be created successfully and Reconciled should be true & InSync should be 'false' for address pools"

Make sure to specify "address pools" in all of the test cases.

Reply:
Changed as requested